pkgs/servers/flaresolverr/default.nix

```diff
@@ -0,0 +1,50 @@
+{ pkgs, lib, stdenv, python3 }:
+with pkgs;
+let
+  python_env = pkgs.python3.withPackages
+    (p: with p; [ bottle waitress selenium func-timeout requests websockets xvfbwrapper webtest ]);
+in
+stdenv.mkDerivation
+rec {
+
+  pname = "flaresolverr";
+  version = "3.0.2";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-zpeJf1CaQ4bsncZz44sH+tFKddYrZf7YdNYL50d9GA4=";
+  };
+
+  buildInputs = [ pkgs.makeWrapper ];
+  nativeBuildInputs = [ chromedriver ];
+  patches = [ ./chromedriver_path.patch ];
+
+  postPatch = ''
+    substituteInPlace src/utils.py \
+    --replace "NIX_STORE_DRIVER_PATH" "${pkgs.chromedriver}/bin/chromedriver"
+  '';
+
+  installPhase = ''
+    mkdir -p $out/share
+    cp -r src $out/share
+    cp package.json $out/share
+  '';
```

Comment:
this should contain `runHook preInstall` and `runHook postInstall` if I’m not mistaken

Reply:
Thanks, done :)